governance-charter.md

```diff
@@ -81,23 +82,22 @@ One year later (in 2020) there will be an election to fill the two seats
 opening up, each with a two year term. The continuity members of the original
 bootstrap committee will continue to serve.
 
-One year after that (2021), the continuity positions will be eliminated and the
-committee will shrink to the final size of 5. There will be an election to fill
-the 3 open seats on the governance committee. The previous continuity members
-are eligible to run for these seats.
+One year after that (2021), there will be an election to fill the 5 open
+seats on the governance committee. The previous continuity members are
+eligible to run for these seats.
 
-The committee will continue to iterate with alternating elections of two and
-three members each year. 
+The committee will continue to iterate with alternating elections of four and
+five members each year. 
 
 For clarity, a table describing this process is given below:
 
 | *Year* | *Continuity* | *Election Cohort #1*    | *Election Cohort #2*    |
 | ------ | ------------ | ----------------------- | ----------------------- |
 | 2019   | 5 people     | **2 people** (2yr term) | **2 people** (1yr term) |
 | 2020   | -            | -                       | **2 people** (2yr term) |
```

Comment:
wait, we need to change it to 4

Reply:
```suggestion
| 2020   | -            | -                       | **4 people** (2yr term) |
```

Reply:
ACK, good catch. I was thinking about the net change, not the number of new people elected. For anyone following along at home, two of the GC members elected in 2019 have a 1-year term (and they are welcome to run for reelection this year).

Reply:
> For anyone following along at home, two of the GC members elected in 2019 have a 1-year term (and they are welcome to run for reelection this year).

Yes, this is an important clarification